gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePlugin.java

```diff
@@ -550,8 +553,8 @@ private static void setupGenericConjureProjects(
                 if (!FIRST_CLASS_GENERATOR_PROJECT_NAMES.contains(conjureLanguage)
                         && !generators.containsKey(conjureLanguage)) {
                     throw new RuntimeException(String.format(
-                            "Discovered subproject %s without corresponding " + "generator dependency with name '%s'",
-                            subproject.getPath(), ConjurePlugin.CONJURE_GENERATOR_DEP_PREFIX + subprojectName));
+                            "Discovered subproject %s without corresponding generator dependency with name '%s'",
+                            subproject.getPath(), ConjurePlugin.CONJURE_GENERATOR_DEP_PREFIX + conjureLanguage));
```

Comment:
not new in this change, but this is an odd mix of string-format and concatenation.

Reply:
yes, there is also inconsistency in the structure of the various pre-checks for the different types of projects.  I originally revamped those to be consistent, but wanted the number of changes to be as small as possible.